src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -180,8 +181,15 @@ def github_comment_workflow(pullrequest_id: int, project_id: int):
         return
 
     # TODO(cathy): add check for OrganizationOption for comment bot
-    if not features.has("organizations:pr-comment-bot", organization):
-        logger.error("github.pr_comment.feature_flag_missing", extra={"organization_id": org_id})
+    if not (
+        features.has("organizations:pr-comment-bot", organization)
```

Comment:
(nit): applying `not` to an `and` statement can be confusing since it's really an `or` statement now, could be more clear (but slightly more verbose) if you just do `not condition_1 or not condition_2`, your call

Reply:
i like using `not (x and y)` because i think the logic is easier to follow in terms of the early return. like "if you don't meet both of these conditions, execute the code block"